include/curl/curl.h

```diff
@@ -57,7 +57,9 @@
  * Define CURL_WIN32 when build target is Win32 API
  */
 
-#if (defined(_WIN32) || defined(__WIN32__) || defined(WIN32)) &&        \
+#if (defined(_WIN32) || \
+     defined(WIN32) /* !checksrc! disable BANNEDPREPROC 1 */ || \
+     defined(__WIN32__)) && \
   !defined(__SYMBIAN32__)
```

Comment:
It's curious that `_WIN32` or `__WIN32__` could end up together with `__SYMBIAN32__`,
so that it had to be guarded-out.

Added in 1960eebc2d021ecf5ffc3f6d4e935d54aa592c72 (2008). Anyhow, Symbian 
was deprecated in curl in 3d64031fa7a80ac4ae3fd09a5939196268b92f81 (2020), so
this no longer a concern.

Reply:
Ref: #12378

Reply:
> It's curious that `_WIN32` or `__WIN32__` could end up together with `__SYMBIAN32__`, so that it had to be guarded-out.

It's been too long for me to recall for certain, but it may have been for x86 builds targeting the Symbian emulator running on Win32.